### ISSUE TITLE
Reverse Engineer not working, The library 'hostpolicy.dll' required to execute the application was not found in 'C:\Program Files\dotnet'.

### ISSUE THREAD

**Issue by MirkoPolimcevic** [ErikEJ/EFCorePowerTools#1019]
I can't pull changes from database. My reverse engineer doesn't work. After open EF Core power tools and select EF Core 5 option I get error: Table list error: A fatal error was encounter. I will add picture of whole error output in attachment. IF I chose EF Core 3 then i get list of pulled tables and procedures, but I cant pull it with it because I am using EF Core 5. My problem started few days ago

### Steps to reproduce
Include a complete description that I can redo to reproduce the issue.
In chapter before I described everything.

### Further technical details
EF Core Power Tools version: (found in About dialog - blue questionmark icon on context menu)
2.5.665
![efCoreError](https://user-images.githubusercontent.com/85869962/121865203-fdb19d00-ccfd-11eb-9cd1-b5d8fde80f74.png)

Database engine: (SQL Server, Azure SQL, Postgres, Oracle, Firebird, SQLite)
SQL Server

Visual Studio version: (e.g. Visual Studio 2019 16.9)
Visual Studio Professional 2019 16.9.6

**Comment by ErikEJ** [comment]
Could you try the latest daily build?

**Comment by ErikEJ** [comment]
I think this could mean your .NET Core 3.1 runtime install is corrupt, you could try a repair/re-install.

**Comment by MirkoPolimcevic** [comment]
Can you tell me what do you mean with latest daily build? I have latest one downloaded from VS Marketplace. I am using .NET 5 and Installed .NET 5 runtime but error stays the same, also reinstalled EF Core Power Tools multiple times but there issues persist. I am adding four more photos. On pic1&2 I am choosing EF Core 3 and I don't get error, but when I chose EF Core 5 I get same error I already attached in my previous post. I have EF Core v5.0.6 in my project, will I get error if I use option for EF Core 3? My coworkers don't have this problem with same project and same version of EF Core Power tools.

**Comment by ErikEJ** [comment]
It sounds like your .net runtime is broken, can you re-install / repair.

Or try another PC?

**Comment by MirkoPolimcevic** [comment]
I tried to repair .net runtime and reinstalled it via Visual Studio Installer and still it don't work. Is there anything other I can try? I can't change current PC. I added pictures in attachment. 


![pic1](https://user-images.githubusercontent.com/85869962/122030550-2b661700-cdce-11eb-8a30-4c037f11a912.jpg)
![pic2](https://user-images.githubusercontent.com/85869962/122030560-2dc87100-cdce-11eb-9873-7b619a4ffaaf.jpg)
![pic3](https://user-images.githubusercontent.com/85869962/122030566-2f923480-cdce-11eb-8f25-372ac6bc6277.jpg)
![pic4](https://user-images.githubusercontent.com/85869962/122030575-315bf800-cdce-11eb-90f7-dd4e4d7c7eea.jpg)
